DebtRunners.py

```diff
@@ -208,25 +196,41 @@ def update(self):
                 self.player.vel.add(vel.normalize().multiply(self.player.speed))
 
 
-class Interaction():
+class Interaction:
     def __init__(self, ):
         pass
 
     def update(self):
         pass
 
 
-class State():
+class State:
     def __init__(self):
         self.start = True
         self.round1 = False
+        self.round2 = False
         self.over = False
 
+    # When the user presses play on the Menu, this should happen. TO BE IMPLEMENTED
     def startGame(self):
-        if self.start:
-            self.start = False
-            self.round1 = True
+        self.start = True
 
+    # Turning the variable that are used to determine which round is currently going, on and off.
+    def round(self, waveNum):
+        # In the beginning I set waveNum to 1, which then sets round1 to true, which will then spawn in the round one enemies.
+        if waveNum == 1:
+            self.round1 = True
+        # After all the enemies from the previous round are gone, are then set that rounds variable to false and the new round variable to true.
+        elif waveNum == 2:
+            self.round1 = False
```

Comment:
if there were 10 rounds this code would be super long, make it into a loop.

Reply:
I think the only way to make is into a loop is update the variable name each time I would go though the loop. I.E second time through I would set self.round1 = False, and on a separate line under that, self.round2 = True. If we would do this in a loop we would have to update the number at the end of self.round, so that next time we go though the loop self.round1 would now be physically changed to self.round2. I'm not sure this is possible to do. There may be away around this, but that may take a little time to implement. So I think this solution if fine for now.